index.js

```diff
@@ -1031,9 +1031,12 @@ Command.prototype.helpInformation = function() {
   if (this._alias) {
     cmdName = cmdName + '|' + this._alias;
   }
+  var ancestor = this
+  var prefix = []
+  while ((ancestor = ancestor.parent)) prefix.push(ancestor.name() + ' ')
```

Comment:
Isn't it better to push just a name of ancestor to array (e.g. `prefix.push(ancestor.name())`) and then...

Reply:
If this has multiple ancestors, e.g. `git remote add`, this will add them in reverse order, so you get `remote git add`